citeproc/source/bibtex/bibtex.py

```diff
@@ -88,8 +90,11 @@ def _bibtex_to_csl(self, bibtex_entry):
                 if value.endswith('+'):
                     value = Pages(first=int(value[:-1]))
                 else:
-                    first, last = value.replace(' ', '').split('--')
-                    value = Pages(first=int(first), last=int(last))
+                    try:
+                        first, last = value.replace(' ', '').split('--')
+                        value = Pages(first=int(first), last=int(last))
```

Comment:
#51 also touches this area, but doesnt quite solve this problem.  Do you recall the data which was needing this fix?